src/components/alt-button/RepostButton.module.css

```diff
@@ -32,6 +32,20 @@
 
 .button {
   all: unset;
+  transform: scale3d(1, 1, 1);
+  transition: all 0.12s ease-in-out;
+}
+
+.button:hover {
```

Comment:
These hover states were being added by the parent before. Moved them here because I assumed we'd want the button to always/by default grow on hover. (same for favorite button)